src/client/game/answer-input.js

```diff
@@ -9,10 +9,17 @@ export default class AnswerInput extends Component {
     componentWillMount() {
         this.setState({
             answerList: [''],
-            focusIndex: 0
+            focusIndex: 0,
+            answerInputTimer: undefined,
```

Comment:
we've changed the timerValue to null as we don't do any checks on undefined for it, however we check whether answerInputTimer is undefined later in the code. Should we change it to null to be consistent?